tests/integration/conftest.py

```diff
@@ -333,7 +323,7 @@ async def fixture_registry_async(
             # Because of this every test must be written in such a way that it can
             # be executed twice with the same servers.
             # "topic_name": new_random_name("topic"),
-            # "group_id": new_random_name("schema_registry")
+            "group_id": new_random_name("registry_async"),
```

Comment:
This is the bug that I started hunting down. `registry_async` and `registry_async_tls` used the same `group_id` but different configs, which caused flakiness on tests when trying to use TLS with the non TLS node.